CMakeLists.txt

```diff
@@ -7,9 +7,6 @@
 # Version 3.15 is the baseline for P4 Control Plane.
 cmake_minimum_required(VERSION 3.15)
 
-# CMAKE_BUILD_TYPE must be set before the first project() command.
```

Comment:
Is this forcing users to specify the build type during compilation (without picking a default?)
A `make-all.sh` should run with some default configuration

Reply:
Ack